consensus/spos/roundConsensus.go

```diff
@@ -47,14 +49,11 @@ func (rcns *roundConsensus) SelfConsensusGroupIndex() (int, error) {
 	return rcns.ConsensusGroupIndex(rcns.selfPubKey)
 }
 
-// EligibleList returns the eligible list ID's
-func (rcns *roundConsensus) EligibleList() []string {
```

Comment:
This method still appears in some tests.

Reply:
that was a different EligibleList